doc/source/user_guide/gotchas.rst

```diff
@@ -367,7 +367,7 @@ integer arrays to floating when NAs must be introduced.
 Differences with NumPy
 ----------------------
 For :class:`Series` and :class:`DataFrame` objects, :meth:`~DataFrame.var` normalizes by
-``N-1`` to produce unbiased estimates of the sample variance, while NumPy's
+``N-1`` to produce unbiased estimates of the population variance, while NumPy's
```

Comment:
why don't we also put the appropriate wiki link here as well.

Reply:
sounds good. for those who are confused with sample and population variance